src/platform/Zephyr/DiagnosticDataProviderImpl.cpp

```diff
@@ -60,6 +64,32 @@ namespace {
 BootReasonType DetermineBootReason()
 {
 #ifdef CONFIG_HWINFO
+
+#ifdef CONFIG_SOC_SERIES_NRF54HX
+    uint32_t reason = nrf_resetinfo_resetreas_global_get(NRF_RESETINFO);
```

Comment:
Any chance to start using `nrf_` specific API, and instead use HW_INFO module or similar? Probably new nrfx which will be part of NCS 2.9.0 introduces support for nRF54H20 and reset reason.

Reply:
Do we know when or how it is planned to be supported?